frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
     "@vitejs/plugin-react": "4.0.4",
+    "esbuild": "0.19.2",
```

Comment:
```suggestion
```

Reply:
I can't run server without these changes, so @v0ldemar01 advise me this in [https://bsa2023.slack.com/archives/C05P563E2MN/p1693412002856019](url)

Reply:
I also had a problem with esbuild, but then I removed all `node_modules` from the project and ran `npm i`. After that everything was fine

Reply:
try to uninstall it, remove and reinstall node_modules